content/en/logs/guide/identify-and-exclude-noisy-logs.md

```diff
@@ -0,0 +1,115 @@
+2. Click the *View All* button in the top right corner to automatically generate the search query associated with this pattern.
+3. Select the `</>` option to the right of the search query and copy the search query.
+4. Navigate to the Configuration page under Logs in the main left hand navigation. Select indexes and click on your associated index. This will populate the option to add an exclusion filter.
+5. Select “Add an Exclusion Filter”.
+6. Input the filter name, define the exclusion query by pasting the search query you copied, and set an exclusion percentage.
+
+Exclusion filters can be disabled at any time by toggling the disable option to the right of the filter. They can also be modified and removed by hovering over the filter and selecting the edit or delete option.
+
+**Note**: If a log matches several exclusion filters, only the first exclusion filter rule is applied. A log is not sampled or excluded multiple times by different exclusion filters.
+
+## 4. Generate metrics to monitor excluded logs
+
+{{< img src="logs/guide/generate-metric.png" alt="Log Patterns View" responsive="true" style="width:75%;">}}
+
+When a subset of logs is excluded from your index, those logs are still available to view in [Live Tail][6]. Moreover, you can generate KPIs from those excluded logs over time at ingest level by creating a new [log-based metric][7]. This metric can then be used for example to create an [anomaly detection][8] monitor to notify you of any unexpected changes.
+
+### Add a new log-based metric
+To generate a new log-based metric based on your log pattern, go to the Logs Configuration page of your Datadog account and select the [Generate Metrics tab][9], then the **New Metric+** button.
+
+1. Under Define Query, input the search query you got from the pattern in step 2.
+2. Select the field you would like to track: Select `*` to generate a count of all logs matching your query or enter a log attribute (e.g., `@network.bytes_written`) to aggregate a numeric value and create its corresponding count, min, max, sum, and avg aggregated metrics.
+3. Add dimensions to group by: Select log attributes or tag keys to apply to the generated log-based metric to transform them into [tags][10] following the `<KEY>:<VALUE>` format. Log-based metrics are considered [custom metrics][11]. Avoid grouping by unbounded or extremely high cardinality attributes like timestamps, user IDs, request IDs, or session IDs to avert impacting your billing.
+4. Name your metric: Log-based metric names must follow the [naming metric convention][12].
+
+### Create an anomaly detection monitor
+Anomaly detection is an algorithmic feature that identifies when a metric is behaving differently than it has in the past. Creating an anomaly detection monitor for your excluded logs will alert you of any changes based on your set alert conditions.
+
+{{< img src="logs/guide/anomaly-alert.png" alt="Log Patterns View" responsive="true" style="width:75%;">}}
```

Comment:
The notification image should be put after the last sentence of this section.

In order to be more relevant? Could we create a "fake metric" `webstore.ok` with an anomaly monitor that drops studently to better emphase the anomaly detection?

Reply:
Updated with all suggestions. Waiting on the fake metric, will add once finalized.